validation/elliptic_solvers/doubly_bounded_poisson.jl

```diff
@@ -0,0 +1,194 @@
+using Oceananigans
+using Oceananigans.Architectures: child_architecture, device
+using Oceananigans.Operators: volume, Δyᶠᶜᵃ, Δyᶜᶠᵃ, Δyᶜᶜᵃ, Δxᶠᶜᵃ, Δxᶜᶠᵃ, Δxᶜᶜᵃ, Δyᵃᶜᵃ, Δxᶜᵃᵃ, Δzᵃᵃᶠ, Δzᵃᵃᶜ, ∇²ᶜᶜᶜ
+using KernelAbstractions: @kernel, @index, Event
+using Oceananigans.Utils: launch!
+using Oceananigans.BoundaryConditions: fill_halo_regions!
+using Oceananigans.Solvers: FFTBasedPoissonSolver, solve!, HeptadiagonalIterativeSolver, constructors, arch_sparse_matrix, matrix_from_coefficients, PreconditionedConjugateGradientSolver, MultigridSolver
```

Comment:
This gives an error because there is no `MultigridSolver`. (Or perhaps there is just nothing pushed yet?)